docs/tutorial/kubernetes-charm-django.rst

```diff
@@ -219,8 +219,8 @@ the architecture of your system:
 
     dpkg --print-architecture
 
-If your host uses ARM64 architecture, open ``rockcraft.yaml`` in a
-text editor and include ``arm64`` under ``platforms``.
+If your host uses ARM architecture, open ``rockcraft.yaml`` in a
+text editor, comment out ``amd64``, and include ``arm64`` under ``platforms``.
```

Comment:
@lengau Can Charmcraft run on ARMhf systems?

Reply:
Technically yes, but the last time snapcraft.io reports a nonzero number of users on armhf (2) was April of last year. One would have to deploy their own juju runners to run charms on armhf AFAIK.